packages/font-downloader/font-downloader.pacscript

```diff
@@ -3,7 +3,7 @@ gives="font-downloader"
 version="10.0.0"
 build_depends="meson libhandy-1-dev gettext"
 # HACK: https://github.com/pacstall/pacstall/issues/727
```

Comment:
```suggestion
```
You are not using any build_depends that don't exist in depends.

Reply:
Actually that got copied from my template ig. Also, that's for a future confirmation. As libhandy-1-dev is a runtime dev too.